examples/with-firebase-hosting/package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "next src/app",
     "serve":
       "echo \"for details see:\n\thttps://github.com/firebase/firebase-tools/issues/535 \n\thttps://github.com/zeit/next.js/issues/3167\";",
-    "deploy": "firebase deploy",
+      "predeploy": "mkdir -p \"dist/functions\"",
```

Comment:
We shouldn't need this as `build-funcs` will create this when run.

Reply:
Yes, but we must run `firebase deploy` in `dist/functions` for fix the problem. If i run `yarn clean` and after `cd \"dist/functions\" && firebase deploy`, we have an error because `dist/functions` not exists.

Reply:
> cd \"dist/functions\" && firebase deploy